Tasks/Common/kubernetes-common-v2/package.json

```diff
@@ -14,7 +14,7 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "azure-pipelines-task-lib": "2.8.0",
-    "vsts-task-tool-lib": "0.4.0",
+    "azure-pipelines-tool-lib": "0.11.0",
```

Comment:
Use `0.12.0` that should update the version in all the other places

Reply:
done